admin/src/main/resources/io/buoyant/admin/js/src/usage.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+define([
+  'jQuery',
+  'Handlebars',
+  'text!template/usage.template',
+], function(
+  $, Handlebars,
+  usageTemplate
+) {
+  return function() {
+    var template = Handlebars.compile(usageTemplate)
+    var $usageContainer = $(".usage-content");
+
+    $.get("admin/metrics/usage").done(function(usageJson) {
+      var view = {usageJson: JSON.stringify(usageJson, null, '  ')};
+      var $templateContainer = $(template(view));
+      $templateContainer.appendTo($usageContainer);
```

Comment:
not that it matters in any way whatsoever, but could be a little less verbose as 
`$usageContainer.html(template(view))`